src/components/search-bar/search-bar.tsx

```diff
@@ -24,6 +24,9 @@ const SearchBar: React.FC<SearchBarProps> = ({
       </label>
       {/* eslint-disable react/jsx-props-no-spreading */}
       <input
+        required
+        pattern=".*\S+.*"
+        title={t("searchNoValidCharactersErrorText")}
```

Comment:
Would it make sense to show a different title if they already typed something in and it's correct? Just a consideration, I think this is also okay 😉

Reply:
"Use the [title](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#title) attribute to specify text that most browsers will display as a tooltip to explain what the requirements are to match the pattern."

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern